spec/models/mailing_spec.rb

```diff
@@ -1,26 +1,40 @@
 require 'spec_helper'
 
 describe Mailing do
-  let(:mailing) { Mailing.new(from: 'from@email.com', to: Role::ROLES,
+  let!(:mailing) { Mailing.new(from: 'from@email.com', to: Role::ROLES,
     cc: 'cc@email.com', bcc: 'bcc@email.com', subject: 'subject', body: '# body') }
 
   it { is_expected.to have_many(:submissions).dependent(:destroy) }
 
   describe '#sent?' do
-    skip 'TODO'
+
+    it 'false before submission' do
```

Comment:
For the sake of readability, I'd suggest to name this *it 'returns false before submission' do*